code/game/objects/items/toys.dm

```diff
@@ -211,6 +211,22 @@
 	attack_verb = list("attacked", "struck", "hit")
 	brightness_on = 0
 
+/obj/item/weapon/twohanded/dualsaber/toy/attack(target as mob, mob/living/user as mob)
+	if(HULK in user.mutations)
+		to_chat(user, "<span class='warning'>You grip the blade too hard and accidentally close it!</span>")
+		unwield()
+		return
+	..()

```

Comment:
So basically this will call the object's parent `/obj/item/weapon/twohanded/dualsaber` and include the CLUMSY check that kills the clown. I'll need to call the parent's parent to bypass that, yes?